tests/test_init_command.rs

```diff
@@ -14,12 +14,18 @@
 
 use std::path::PathBuf;
 
+use test_case::test_case;
+
 use crate::common::TestEnvironment;
 
 pub mod common;
 
-fn init_git_repo(git_repo_path: &PathBuf) {
-    let git_repo = git2::Repository::init(git_repo_path).unwrap();
+fn init_git_repo(git_repo_path: &PathBuf, bare: bool) {
+    let mut opts = git2::RepositoryInitOptions::new();
+    if bare {
+        opts.bare(true);
+    }
```

Comment:
nit: just simply set `opts.bare(bare)`?

Reply:
While at that just inlined it to avoid mut, though not sure with `cargo fmt` it's actually better

Reply:
I don't think so, opts.bare returns &mut, and something needs to keep the target alive across statements. Unless I'm missing some language trick, but

```
let opts = RepositoryInitOptions::new().bare(bare);
```

Doesn't compile because it refers to a temporary value.

Reply:
Ah, you're right. I'm surprised it even works when the expression is inlined.